recipes/dcw-gmt/build.bat

```diff
@@ -0,0 +1,8 @@
+set DATADIR="%LIBRARY_PREFIX%\share\gcw"
```

Comment:
Package name is wrong here as well. Should be: `set DATADIR="%LIBRARY_PREFIX%\share\dcw-gmt"`